### DISCUSSION TITLE
error in roboshop ansible

### DISCUSSION THREAD

**Discussion by LavanyaRamesh319** [learndevopsonline/learndevopsonline#1894] [Q&A]
I am getting error while i am running catalogue component.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/103929459/2d9fa916-ae5a-4537-b354-0ec5525234f9)

**Comment by chaithanya1812** [comment]
Hi @LavanyaRamesh319, 
### I didn't find actual solution for this error, your servers are not reachable.. 
## so I try with my instance for me it's working fine...
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/f3ae8870-f951-4b61-b9f1-a0fc38218d82)

## IMP:- Hi sir, I found out why Jenkins server having slow response. every time we stop & start  the jenkins server we will get new public-ip ,we need to take this new public-ip & go manage Jenkins settings we need to this configuration..
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/4dce39cb-d5de-402e-9d15-6cee2f35afc3)

**Comment by LavanyaRamesh319** [comment]
Thank you @chaithanya1812 It got resolved
